kbase-extension/static/kbase/js/common/jobCommChannel.js

```diff
@@ -329,100 +328,82 @@ define([
+                            jobId: msgData[_jobId].job_id,
+                            jobInfo: msgData[_jobId],
+                        });
                     });
                     break;
 
-                case 'job_info_batch':
-                    // TODO: awaiting backend implementation
-                    break;
-
-                case 'job_init_err':
-                case 'job_init_lookup_err':
-                    this.displayJobError(msgData);
-                    console.error('Error from job comm:', msg);
-                    break;
-
-                case 'job_logs':
-                    this.sendBusMessage(JOB, msgData.job_id, 'job-logs', {
+                case BACKEND_RESPONSES.LOGS:
+                    this.sendBusMessage(JOB, msgData.job_id, RESPONSES.LOGS, {
                         jobId: msgData.job_id,
                         logs: msgData,
                         latest: msgData.latest,
                     });
                     break;
 
-                case 'jobs_retried':
+                case BACKEND_RESPONSES.RETRY:
                     msgData.forEach((jobRetried) => {
-                        this.sendBusMessage(
-                            JOB,
-                            jobRetried.job.state.job_id,
-                            'job-retry-response',
-                            {
-                                job: {
-                                    jobState: jobRetried.job.state,
-                                    outputWidgetInfo: jobRetried.job.widget_info,
-                                },
-                                retry: {
-                                    jobState: jobRetried.retry.state,
-                                    outputWidgetInfo: jobRetried.retry.widget_info,
-                                },
-                            }
-                        );
+                        this.sendBusMessage(JOB, jobRetried.job.state.job_id, RESPONSES.RETRY, {
+                            job: {
+                                jobState: jobRetried.job.state,
+                                outputWidgetInfo: jobRetried.job.widget_info,
+                            },
+                            retry: {
+                                jobState: jobRetried.retry.state,
+                                outputWidgetInfo: jobRetried.retry.widget_info,
+                            },
+                        });
                     });
                     break;
 
                 /*
                  * The job status for one or more jobs. See job_status_all
                  * for a message which covers all active jobs.
-                 * Note that these messages are additive to the job panel
-                 * cache, but the reverse logic does not apply.
+                 *
+                 * data structure: object with key jobId and value { jobState: job.state, outputWidgetInfo: job.widget_info }
                  */
-                case 'job_status':
```

Comment:
there were a number of backend responses that returned job statuses; they all now use the message type `job_status`